src/turbolinks/cache.coffee

```diff
@@ -9,12 +9,12 @@ class Turbolinks.Cache
 
   get: (location) ->
     return unless @has(location)
-    snapshot = @read(location)
+    snapshot = @read(location).clone()
```

Comment:
[Aha, nvm, found your old justification here, which makes sense to me.](https://github.com/turbolinks/turbolinks-classic/pull/24#issuecomment-8944174)